.github/workflows/release.yaml

```diff
@@ -5,7 +5,6 @@ name: Release to Charmhub
 on:
   push:
     branches:
-      - 3/edge
```

Comment:
I don't think we need this here anymore? The two branches are diverging and the refresh requirements also break CI consistency